app/views/grades/_teams.html.erb

```diff
@@ -32,7 +32,7 @@
 </thead>
 <!--E1877: tbody tag added to the table structure-->
 <tbody>
-<% 0.upto(@scores[:teams].length - 1).each do |index| %>   
```

Comment:
Does this fix #2258 ?

Reply:
It's not ready to merge. I still need to apply the late calculation to the score